relayer/local_relayer.go

```diff
@@ -117,6 +117,11 @@ func (r *LocalRelayer) GetNonce(ctx context.Context, walletConfig core.WalletCon
 	return sequence.GetWalletNonce(r.GetProvider(), walletConfig, walletContext, space, blockNum)
 }
 
+func (r *LocalRelayer) Simulate(ctx context.Context, txs *sequence.SignedTransactions) ([]*sequence.RelayerSimulateResult, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
I assume we don't need to implement this?

Reply:
Not for now. If we want to use it locally then probably yes.